### ISSUE TITLE
Cannot access individual expand and collapse buttons in write plan 

### ISSUE THREAD

**Issue by mariapraetzellis** [CDLUC3/dmptool#375]
Applicable WCAG Standard(s)
[2.1.1 Keyboard](http://www.w3.org/TR/WCAG20/#keyboard-operation-keyboard-operable) (Level A)
